dev/bots/test.dart

```diff
@@ -718,6 +720,7 @@ Future<void> _pubRunTest(String workingDirectory, {
   bool useBuildRunner = false,
   String coverage,
   bq.TabledataResourceApi tableData,
+  bool forceSingleCore = false,
```

Comment:
Without this change, we would try to spin up multiple chrome instances and stall. I'm also suspicious that some of the previous flakes in integration tests were caused by having multiple flutter_tester shells active.

Reply:
This should go in a comment.

Reply:
This is interesting. Do you not use the concurrency parameter of `package:test`?

Reply:
We do, but by default it gets set to the number of cores that the bot has.